test/test_functions.py

```diff
@@ -444,7 +445,7 @@ def test_toboolean(self):
     #         Query().where(f.todatetime('') > datetime.datetime(2019, 7, 23)).render(),
     #         " | where (startofday(foo)) > datetime(2019-07-23 00:00:00.000000)")
     # def todatetime(expr: StringType) -> DatetimeExpression:
-    #     return DatetimeExpression(KQL('todatetime({})'.format(expr)))
+    #     return DatetimeExpression(KQL('todatetime({})'.format(_subexpr_to_kql(expr))))
```

Comment:
why is this part commented? don't we need to check todatetime too?

Reply:
It was already commented